Source/Extensions/Blazorise.Charts.DataLabels/wwwroot/chart.datalabels.js

```diff
@@ -0,0 +1,44 @@
+import { getChart } from "../Blazorise.Charts/charts.js?v=1.1.2.0";
+
+// Register the plugin to all charts:
+Chart.register(ChartDataLabels);
```

Comment:
This line isn't wrapped in any way. For example in a document ready state, any way it can run when it should not?

Reply:
You're probably looking at an older commit. This is not true anymore.